<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOptSetup</class>
 <widget class="QDialog" name="DialogOptSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>653</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Optimization Setup</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QPushButton" name="btnRemoveTarget">
       <property name="text">
        <string>Remove target</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblParametersLabel">
       <property name="text">
        <string>Parameters:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblCompartmentLabel">
       <property name="text">
        <string>Population:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnEditTarget">
       <property name="text">
        <string>Edit target</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QListWidget" name="lstParameters"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QListWidget" name="lstTargets"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnAddTarget">
       <property name="text">
        <string>Add target</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblTargetsLabel">
       <property name="text">
        <string>Targets:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btnEditParameter">
       <property name="text">
        <string>Edit parameter</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="btnAddParameter">
       <property name="text">
        <string>Add parameter</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="btnRemoveParameter">
       <property name="text">
        <string>Remove parameter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSpinBox" name="spinPopulation">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinPopulation</tabstop>
  <tabstop>lstTargets</tabstop>
  <tabstop>btnAddTarget</tabstop>
  <tabstop>btnEditTarget</tabstop>
  <tabstop>btnRemoveTarget</tabstop>
  <tabstop>lstParameters</tabstop>
  <tabstop>btnAddParameter</tabstop>
  <tabstop>btnEditParameter</tabstop>
  <tabstop>btnRemoveParameter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
